azavea/raster-foundry
# Fix imports view dropdown menus

## Overview

Fix the dropdown menu on the imports view

### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

### Demo

![image](https://cloud.githubusercontent.com/assets/4392704/26367138/966e9d74-3fbb-11e7-9869-8775cdfe2b8c.png)


### Notes

@designmatty The z-index context was really messed up from all the `position: relatives` which didn't seem to actually do anything, so I tore them all out, and as far as I can tell it hasn't broken anything other than the ingest tag which I fixed. I'd like your feedback on the css.

## Testing Instructions

* Verify that in places where we show an ingest status tag, it doesn't look messed up
* Verify that the dropdown menu is no longer behind other elements

Closes #1689
